### DISCUSSION TITLE
How do I configure just a single target?

### DISCUSSION THREAD

**Discussion by smac89** [xmake-io/xmake#5845] [Q&A] [answered]
When you type `xmake f -h`, the usage example includes `target` as the last option:

```
Usage: $xmake config|f [options] [target]
```

However, supplying a target option leads to the command erroring with:

```
error: invalid argument: target_name
```

Is this option actually supported or not?

**Comment by waruqi** [comment, marked as answer]
Sorry, it's a deprecated option. and it has been removed. This usage is incorrect.  I have updated it on dev.

https://github.com/xmake-io/xmake/commit/cefc6b3f0f70508da1f7a64d7d2ad64e45be036c
